keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -302,13 +314,27 @@ type TopologyHint struct {
     Preferred bool
 }
 
-// HintProvider is implemented by Kubelet components that make
-// topology-related resource assignments. The Topology Manager consults each
-// hint provider at pod admission time.
+// HintProvider is an interface for components that want to collaborate to
+// achieve globally optimal concrete resource alignment with respect to
+// NUMA locality.
```

Comment:
You shouldn't limit the description to NUMA locality. This happens to be the only constraint currently, but we plan to add more going forward.